app/src/views/search-engine/components/articles-list/styles.js

```diff
@@ -0,0 +1,19 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Is it possible with this library to read the CSS from a .css file?

Google style guide recommends putting CSS/styling in its own file: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Seems like React is for separating it into .css files too:
https://reactjs.org/docs/faq-styling.html#what-is-css-in-js

Reply:
Yeah, this way is based of Material UI way of [encapsulating classes](https://material-ui.com/styles/api/#makestyles-styles-options-hook). I was just following Material UI way of using CSS classes.